tinyagentos/routes/app_permissions.py

```
@@ -110,3 +111,55 @@ async def revoke_app_permission(
+    notifs = getattr(request.app.state, "notifications", None)
+    if notifs is not None:
+        # Best effort: a notification failure must not fail the queued decision.
+        try:
+            await notifs.add(
+                title="Permission needed",
+                message=f"{app_id} is requesting permissions",
+                level="warning",
+                source="decisions",
+            )
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)